src/notifier/__init__.py

```diff
@@ -40,7 +40,8 @@ class EventService(Enum):
     HARVESTER = 0
     FARMER = 1
     FULL_NODE = 2
-    DAILY = 3
+    WALLET = 3
```

Comment:
Probably not really an issue at this time, but it might be better to append rather than change the values of an enum, considering this may be a breaking change for integrations at some point.

Reply:
Good catch! The order only got shuffled like that because I did the features in reverse order the first time.